avaframe/com2AB/com2AB.py

```diff
@@ -247,32 +249,32 @@ def findSplitPoint(AvaProfile, splitPoint):
 
 
 
-def readABinputs(cfgINPATH):
-    cfgAva = cfgINPATH['AvaLancheRoot']
+def readABinputs(cfgAva):
+
     cfgPath = {}
 
-    ProfileLayer = glob.glob(cfgAva + '/Inputs/LINES/*.shp')
-    cfgPath['ProfileLayer'] = ''.join(ProfileLayer)
+    profileLayer = glob.glob(cfgAva + '/Inputs/LINES/*.shp')
+    cfgPath['profileLayer'] = ''.join(profileLayer)
 
-    DGMSource = glob.glob(cfgAva + '/Inputs/*.asc')
+    dgmSource = glob.glob(cfgAva + '/Inputs/*.asc')
     try:
-        assert len(DGMSource)==1, 'There should be only one and only one DEM .asc file in ' + cfgAva + '/Inputs/'
+        assert len(dgmSource)==1, 'There should be only one DEM .asc file in ' + cfgAva + '/Inputs/'

```

Comment:
There should be one and only one DEM... This message stands if there is no DEM file.
We should use DEM and not DGM i think